src/js/tls.js

```diff
@@ -212,10 +197,9 @@ function createServer(options, secureConnectionListener) {
 
 function connect(arg0, arg1, arg2, callback) {
   var options;
-  var tlsSocket;
   if (typeof arg0 == 'object') {
     options = Object.create(arg0, {
-      isServer: { value: false, enumerable: true },
+      isServer: { value: arg0.isServer || false, enumerable: true },
```

Comment:
`|| false` is pointless. Just let it be `arg0.isServer`.

Reply:
I just realiezd I don't understand this. How can a server connects???

Reply:
I left that in by accident, it doesn't have to be there